key.core/src/test/java/de/uka/ilkd/key/proof/reference/TestReferenceSearcher.java

```diff
@@ -104,10 +104,10 @@ void checksUserLemmas() throws Exception {
                 "proofCaching/proofWithRule.proof"));
         Proof p3 = env3.getLoadedProof();
 
-        DefaultListModel<Proof> previousProofs = new DefaultListModel<>();
-        previousProofs.addElement(p);
-        DefaultListModel<Proof> newProof = new DefaultListModel<>();
-        newProof.addElement(p2);
+        List<Proof> previousProofs = new CopyOnWriteArrayList<>();
+        previousProofs.add(p);
+        List<Proof> newProof = new CopyOnWriteArrayList<>();
+        newProof.add(p2);
```

Comment:
`newProof` is never read; can it be removed?

Reply:
Yes. (I wrote this test)